<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>obsProjectConfigDialog</class>
 <widget class="QDialog" name="obsProjectConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>239</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OBS Project Configuration</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="projectNameLabel">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="projectNameLineEdit"/>
   </item>
   <item row="4" column="1">
    <widget class="QDialogButtonBox" name="obsProjectConfigButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="projectTargetComboBox">
     <property name="editable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="projectTargetLabel">
     <property name="text">
      <string>Target:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="projectArchitectureComboBox"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="projectArchitectureLabel">
     <property name="text">
      <string>Architecture:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="projectServerLabel">
     <property name="text">
      <string>OBS Server:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="projectServerComboBox"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>obsProjectConfigButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>obsProjectConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>obsProjectConfigButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>obsProjectConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
